src/runtime_src/core/edge/user/shim.h

```diff
@@ -271,6 +271,12 @@ class shim {
 
   void
   hwctx_exec_buf(const xrt_core::hwctx_handle* hwctx_hdl, xclBufferHandle boh);
+
+  int
+  open_graph_context(const zynqaie::hwctx_object* hwctx, const uuid_t xclbinId, unsigned int graph_id, xrt::graph::access_mode am);
```

Comment:
```suggestion
  void
  open_graph_context(const zynqaie::hwctx_object* hwctx, const uuid_t xclbinId, unsigned int graph_id, xrt::graph::access_mode am);
```

Reply:
hi, marking the conversation as resolved as I pushed the changes.